src/CMakeLists.txt

```diff
@@ -50,9 +51,9 @@ IF (GIT_FOUND AND IS_GIT_PROJECT)
     COMMAND ${CMAKE_COMMAND} -E echo_append "#define GIT_COMMIT_LAST_CHANGED_RAW " >> ${GITREV_TMP}
     COMMAND ${GIT_EXECUTABLE} log -1 --format=%ad >> ${GITREV_TMP}
     COMMAND ${CMAKE_COMMAND} -E echo >> ${GITREV_TMP}
-    COMMAND ${CMAKE_COMMAND} -E echo_append "#define GIT_COMMIT_SUBJECT_RAW " >> ${GITREV_TMP}
-    COMMAND ${GIT_EXECUTABLE} log -1 --format="%s" >> ${GITREV_TMP}
-    COMMAND ${CMAKE_COMMAND} -E echo >> ${GITREV_TMP}
+    COMMAND ${CMAKE_COMMAND} -E echo_append "#define GIT_COMMIT_SUBJECT_RAW \"" >> ${GITREV_TMP}
+    COMMAND ${GIT_EXECUTABLE} log -1 --format=%s | sed -E "s/\"/\\\"/g" | tr -d '\\n' >> ${GITREV_TMP}
```

Comment:
To make this work my Ubuntu 14.04 box, I need to add a couple more backslashes to the sed replacement:  sed -E "s/\"/\\\\\"/g"